### PR TITLE
Ready for multiple runs

### PR DESCRIPTION
**Pull request by aferron**
Edits cleanify.py with an ID for the run labeled as "distortion" but it could be used to keep track of any parameter set.
Labels Saved_Images_<ID#> files with that ID as well as all the images in the file.
Image name is 
<Clean/Cleaned/Dirty> _ <Epoch number (e#)> _ <number within the set (i#)> _ <ID number (d#)>

Also includes small edits to the CNN

### CODE REVIEW

File: cleanify.py [hunk 1]
```diff
@@ -203,15 +207,15 @@ def forward(self, x):
...
+def fit(model, dataloader):
 
     # Set the model in training mode
     model.train()
     running_loss = 0.0
 
     # i doesn't get used in this for loop
     # tqdm wraps around an interable to make a progress bar
-    for i, data in tqdm(enumerate(dataloader), total=len(train_data) // dataloader.batch_size):
+    for data in tqdm(dataloader, total=int(len(train_data)/dataloader.batch_size)):
```
Comment (aferron) [inline cleanify.py:218]: This was probably a git version thing
I didn't change it back to int()/

**Review by howlium — APPROVED** [review]
